.github/workflows/backup.yml

```diff
@@ -0,0 +1,29 @@
+name: AWS SSM Send-Command Backup
+
+on: 
+  workflow_dispatch:
+
+jobs:
+  start:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: AWS SSM Send-Command
+        uses: peterkimzz/aws-ssm-send-command@master
+        id: ssm
+        with:
+          aws-region: ${{ secrets.AWS_REGION }}
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          instance-ids: ${{ secrets.INSTANCE_ID }}
+
+          #TODO: update these values to point to the backup script
+          working-directory: /home/ubuntu/application
+          command: ls -al #something like -> /bin/sh backup.sh
+          comment: Hello world!
```

Comment:
Change to:
```
working-directory: /home/ec2-user/
command: ./stop_backup_start.sh
comment: StopBackupStart
```

The script we're using is here: https://github.com/rcatlin/valheim-server-cdk/blob/master/templates/stop_backup_start.sh.hbs